free-courses-th.md

```diff
@@ -0,0 +1,65 @@
+### Index
+
+* [C](#c)
+* [C#](#c-sharp)
+* [C++](#cpp)
+* [Git](#git)
+* [JavaScript](#javascript)
+  * [Angular.js](#angularjs)
+  * [NodeJS](#nodejs)
+  * [React](#react)
+  * [Vue.js](#vuejs)
+* [Python](#python)
+
+
+### C
+
+* [บทเรียนภาษา C](http://marcuscode.com/lang/c) (MarcusCode)
+
+
+### C-Sharp
+
+* [บทเรียนภาษา C#](http://marcuscode.com/lang/csharp) (MarcusCode)
+
+
+### CPP
+
+* [บทเรียนภาษา C++](http://marcuscode.com/lang/cpp) (MarcusCode)
+
+
+### Git
+
+* [มาเรียนรู้ Git แบบง่ายๆกันเถอะ](https://blog.nextzy.me/มาเรียนรู้-git-แบบง่ายๆกันเถอะ-427398e62f82)  (Ake Exorcist)
+* [สอน git และ github เบื้องต้น](https://www.youtube.com/playlist?list=PLoTScYm9O0GGsV1ZAyP4m_iyAbflQrKrX)  (prasertcbs)
+* [สอนใช้ Git - Version Control System](https://www.youtube.com/playlist?list=PLjPfp4Ph3gBrgVPZySWHZwxXSxdgOKhQ-)  (CMDev)
+
+
+### JavaScript
+
+* [จาวาสคริปต์เบื้องต้น](https://phyblas.hinaboshi.com/saraban/javascript) (Phyblas)
+* [สอน JavaScript](https://www.youtube.com/playlist?list=PL_xSQKvnccplgKmdtqizMGRh11witheTM)  (Zinglecode)
+* [บทเรียนภาษา Javascript](http://marcuscode.com/lang/javascript) (MarcusCode)
```

Comment:
On the website the resource is called: `ภาษา JavaScript` on the website.
Could we please use the name as described there?

Reply:
Could you please write `JavaScript` with a capital S, the same way that it is on the website?
Java**S**cript is the correct spelling for it anyway.